### DISCUSSION TITLE
amamba/quickstart/pipeline-on-node/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#4991] [Announcements]
# amamba/quickstart/pipeline-on-node/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/amamba/quickstart/pipeline-on-node/

**Comment by san3Xian** [comment]
注意默认template中就有affinity定义，但是 是prefer级别弱绑定
